### DISCUSSION TITLE
healthcheck endpoint

### DISCUSSION THREAD

**Discussion by ShlomiPorush** [docusealco/docuseal#186] [General]
Hi. on version 1.3.6 the changelog:
🆕 Add healthcheck endpoint

does anyone know what is the url? I want to monitor it via external system.

**Comment by omohokcoj** [comment]
@ShlomiPorush try `/up` endpoint

**Reply by ShlomiPorush** [reply]
thanks. I get an empty page.
A green page to be precise

**Reply by omohokcoj** [reply]
@ShlomiPorush https://docuseal.co/up - that's expected behavior - if the server responds with 200 then the app is running

**Reply by ShlomiPorush** [reply]
I thought it was a metrics page.
